### DISCUSSION TITLE
✅ How do I pay for this?

### DISCUSSION THREAD

**Discussion by DavidFW1960** [isontheline/pro.webssh.net#320] [General]
It's impossible! In app purchases just goes to versions page and I can't purchase it. I think I got one option to purchase but that was only starter and I'd like to get pro

**Comment by isontheline** [comment]
Hello @DavidFW1960 👋

WebSSH 14.18.1 as a "glitch" (but In App Purchases works) :
* The glitch : You can only buy STARTER or PRO when you add a new connection and you can't purchase them from the settings

The glitch has been fixed in 14.19 TestFlight release : https://testflight.apple.com/join/QSrBK59z

If you think I didn't understand your issue, please post a screenshot.

Sorry for the trouble.

**Comment by DavidFW1960** [comment]
Well no in-app purchases just shows me the comparison page between the 3 versions and I did get a purchase screen when I added a new connection but it only showed me the Starter version, not pro.

**Comment by DavidFW1960** [comment]
So I added another connection and no purchase prompt. Loaded TestFlight version and no in-app-purchase option I can see....

**Reply by isontheline** [reply]
So you already have PRO (aka Unlimited) IAP 😉

When do you have downloaded WebSSH for the first time?

**Comment by DavidFW1960** [comment]
Downloaded 30 Aug 2020..

**Reply by isontheline** [reply]
Yes if you have downloaded it before 14.2 (31 august 2020) then you have PRO / UNLIMITED already inside it for **free**.

Have a nice day☀️

**Comment by DavidFW1960** [comment]
Wow! So I made it by 1 day.... hahaha. Thanks and you have a great day as well. This is a great app. Maybe I saw a posting about it being free till x date and downloaded it then.
Anyway I really appreciate this app and now your excellent support as well. Perhaps the about function in the app could tell you what licence you have in future?

**Reply by isontheline** [reply]
Thank you a lot for supporting WebSSH 👍
I hope you will enjoy the upcoming releases 😇

You're right I need to improve the about dialog. I've just created an issue referencing your comment : https://github.com/isontheline/pro.webssh.net/issues/321
